src/Pyz/Yves/CheckoutPage/Theme/default/views/address/address.twig

```diff
@@ -45,63 +65,76 @@
                 {% endif %}
             </h2>
 
-            {% if hasShippingAddresses %}
+            {% if shippingAddress is defined %}
```

Comment:
You will have an error on the variable initialization.

Reply:
Reverted